src/Tizen.NUI/src/internal/WebBackForwardList.cs

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2017 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// WebBackForwardListItem
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardListItem : Disposable
+    {
+        /// <summary>
+        /// Get uri.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Url
+        {
+            get
+            {
+                return Interop.WebBackForwardListItem.GetUrl(swigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Get title.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Title
+        {
+            get
+            {
+                return Interop.WebBackForwardListItem.GetTitle(swigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Get original url.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string OriginalUrl
```

Comment:
Url, OriginalUrl -> CA1056/CA1054 warnings
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1056

Could you refer this patch?
https://github.com/Samsung/TizenFX/pull/2364

Reply:
Done.